### PR TITLE
Move etcd tools to components

### PR DESCRIPTION
**Pull request by polskikiel**
**Description**

Changes proposed in this pull request:

- Moved etcd tools to components

**Related issue(s)**

### CODE REVIEW

File: development/tools/jobs/kyma/etcd_backup_job_test.go [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package tools_test
+package kyma
```
Comment (jasiu001) [inline development/tools/jobs/kyma/etcd_backup_job_test.go]: each test in `/.../jobs/kyma` directory has package name `kyma_test`, to be consistent it also should have that name

File: development/tools/jobs/kyma/etcd_tls_setup_job_test.go [hunk 2]
```diff
@@ -1,4 +1,4 @@
-package tools_test
+package kyma
```
Comment (jasiu001) [inline development/tools/jobs/kyma/etcd_tls_setup_job_test.go]: It should be `kyma_test` like rest of the files

File: prow/jobs/kyma/components/etcd-backup-job/etcd-backup-job.yaml [hunk 3]
```diff
@@ -16,7 +16,7 @@ job_template: &job_template
         command:
```
Comment (jasiu001) [inline prow/jobs/kyma/components/etcd-backup-job/etcd-backup-job.yaml]: What about job name, now is `pre-master-kyma-tools-etcd-backup` maybe should be `pre-master-kyma-tools-etcd-backup-job`?

File: development/tools/jobs/kyma/etcd_backup_job_test.go (hunk 1 above)
Comment (polskikiel) [reply development/tools/jobs/kyma/etcd_backup_job_test.go]: changed

File: development/tools/jobs/kyma/etcd_tls_setup_job_test.go (hunk 2 above)
Comment (polskikiel) [reply development/tools/jobs/kyma/etcd_tls_setup_job_test.go]: done

File: prow/jobs/kyma/components/etcd-backup-job/etcd-backup-job.yaml (hunk 3 above)
Comment (polskikiel) [reply prow/jobs/kyma/components/etcd-backup-job/etcd-backup-job.yaml]: i changed both jobs names:
`pre-master-kyma-tools-etcd-backup` --> `pre-master-kyma-components-etcd-backup-job`
`pre-master-kyma-tools-etcd-tls-setup` --> `pre-master-kyma-components-etcd-tls-setup-job`

**Pull request merged**